src/jwplayer/utils/iasTracker.ts

```diff
@@ -0,0 +1,25 @@
+import { scriptLoader } from 'jwplayer/utils/scriptLoader';
+
+const scriptUrl = '//static.adsafeprotected.com/vans-adapter-google-ima.js';
+class IasTracker {
+	private scriptPromise: Promise<Event>;
+
+	public async loadIasTrackerIfEnabled(skin: 'fandomdesktop' | 'fandommobile'): Promise<void> {
```

Comment:
You mean to create a separate util to parse the config?

Reply:
I meant that the tracker should probably remain unaware of the FCP, its various skins, or the user's device. This is important since our players are also used on other platforms and sites, right?

After some consideration, I suggest using the full config item key instead of a skin name. It would make it clearer if the IAS Tracker's status is based on the instant config key rather than on a specific 'skin'. What are your thoughts on this? Here's a quick suggestion for how it might look:

```suggestion
	public async loadIasTrackerIfEnabled(instantConfigKey: string): Promise<void> {
```